### PR TITLE
 net-im/yowsup: bump to version 3.2.3

### PR DESCRIPTION
**Pull request by ConiKost**
This is a new major release of net-im/yowsup,
which needs new dependencies:
dev-python/consonance
dev-python/dissononce
dev-python/transitions

Package-Manager: Portage-2.3.69, Repoman-2.3.16
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

### CODE REVIEW

File: dev-python/dissononce/dissononce-0.34.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A python implementation for Noise Protocol Framework"
+HOMEPAGE="https://github.com/tgalal/dissononce"
+SRC_URI="https://github.com/tgalal/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```
Comment (juippis) [inline dev-python/dissononce/dissononce-0.34.3.ebuild]: This looks wrong, I see MIT in their github page?

File: dev-python/transitions/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		A lightweight, object-oriented state machine implementation in Python.
```
Comment (juippis) [inline dev-python/transitions/metadata.xml]: This isn't any longer than the default ebuild description.

**Comment by juippis** [comment]
Also remember that `RESTRICT="!test? ( test )"` thingie to those ebuilds that dont have test restricted.

**Comment by juippis** [comment]
```
>>> Test phase: dev-python/dissononce-0.34.3
 * python2_7: running distutils-r1_run_phase python_test
python2.7 setup.py test
running test
running egg_info
creating dissononce.egg-info
writing requirements to dissononce.egg-info/requires.txt
writing dissononce.egg-info/PKG-INFO
writing top-level names to dissononce.egg-info/top_level.txt
writing dependency_links to dissononce.egg-info/dependency_links.txt
writing manifest file 'dissononce.egg-info/SOURCES.txt'
reading manifest file 'dissononce.egg-info/SOURCES.txt'
writing manifest file 'dissononce.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
 * python3_5: running distutils-r1_run_phase python_test
python3.5 setup.py test
running test
running egg_info
writing requirements to dissononce.egg-info/requires.txt
writing dissononce.egg-info/PKG-INFO
writing dependency_links to dissononce.egg-info/dependency_links.txt
writing top-level names to dissononce.egg-info/top_level.txt
reading manifest file 'dissononce.egg-info/SOURCES.txt'
writing manifest file 'dissononce.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.001s

OK
 * python3_6: running distutils-r1_run_phase python_test
python3.6 setup.py test
running test
running egg_info
writing dissononce.egg-info/PKG-INFO
writing dependency_links to dissononce.egg-info/dependency_links.txt
writing requirements to dissononce.egg-info/requires.txt
writing top-level names to dissononce.egg-info/top_level.txt
reading manifest file 'dissononce.egg-info/SOURCES.txt'
writing manifest file 'dissononce.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.001s

OK
 * python3_7: running distutils-r1_run_phase python_test
python3.7 setup.py test
running test
running egg_info
writing dissononce.egg-info/PKG-INFO
writing dependency_links to dissononce.egg-info/dependency_links.txt
writing requirements to dissononce.egg-info/requires.txt
writing top-level names to dissononce.egg-info/top_level.txt
reading manifest file 'dissononce.egg-info/SOURCES.txt'
writing manifest file 'dissononce.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.001s

OK
 * python2_7: running distutils-r1_run_phase _distutils-r1_clean_egg_info
 * python3_5: running distutils-r1_run_phase _distutils-r1_clean_egg_info
 * python3_6: running distutils-r1_run_phase _distutils-r1_clean_egg_info
 * python3_7: running distutils-r1_run_phase _distutils-r1_clean_egg_info
>>> Completed testing dev-python/dissononce-0.34.3
```
does this actually do anything?

File: dev-python/transitions/metadata.xml (hunk 2 above)
Comment (ConiKost) [reply dev-python/transitions/metadata.xml]: Added some additional text.

File: dev-python/dissononce/dissononce-0.34.3.ebuild (hunk 1 above)
Comment (ConiKost) [reply dev-python/dissononce/dissononce-0.34.3.ebuild]: Good catch, I must overseen this. Changed.

**Comment by ConiKost** [comment]
> Also remember that RESTRICT="!test? ( test )" thingie to those ebuilds that dont have test restricted.

Added.

> does this actually do anything?

I asked upstream. Currently, no tests are there, but will be added in future. Should be still leave it, so it will automatically trigger on version bump?

**Comment by juippis** [comment]
> I asked upstream. Currently, no tests are there, but will be added in future. Should be still leave it, so it will automatically trigger on version bump?

Yeah, I'd RESTRICT="test" now and add some comment above it saying to always check on new release.
```
# Check after every release
RESTRICT="test"
```

I can add it, or you can if you wish. Just let me know. Rest looks good!

**Comment by ConiKost** [comment]
> I can add it, or you can if you wish. Just let me know. Rest looks good!

Fine for me, added.

File: dev-python/transitions/transitions-0.6.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+RDEPEND="
+	dev-python/pygraphviz[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	${RDEPEND}
+	test? (
+		dev-python/pycodestyle[${PYTHON_USEDEP}]
```
Comment (juippis) [inline dev-python/transitions/transitions-0.6.9.ebuild]: Please sort this when touching the ebuild next time :)

**Pull request closed without merging**